src/database/index.tsx

```diff
@@ -19,7 +19,7 @@ import { prepareForStorage, prepareFromStorage } from "./utils";
 const tokenExchangeEndpoint =
   "https://us-central1-c19-backend.cloudfunctions.net/getFirebaseToken";
 const modelInputsCollectionId = "model_inputs";
-const scenariosCollectionId = "scenarios";
+const scenariosCollectionId = "scenarios_test";
```

Comment:
I used a separate parent collection while testing, but this is reset in the final [clean up commit](https://github.com/Recidiviz/covid19-dashboard/pull/223/commits/aa2bb0593d5153a5473fc15a8b07f3ec1df2c8ac).